projects/Amlogic_Legacy/linux/linux.aarch64.conf

```diff
@@ -2431,7 +2431,8 @@ CONFIG_MEDIA_USB_SUPPORT=y
 #
 # Webcam devices
 #
-# CONFIG_USB_VIDEO_CLASS is not set
+CONFIG_USB_VIDEO_CLASS=m
+# CONFIG_USB_VIDEO_CLASS_INPUT_EVDEV is not set
```

Comment:
@nicolaspernoud could you drop the changes at `projects/Amlogic_Legacy/linux/linux.aarch64.conf` 

due the nature of that OLD codebase it is very likely it creates some random problems that I would like to avoid, we are at very low maintenance status for the old AML kernel so every change could cause problems nobody ever fixes

the benefits of that changes at these kernel is near zero so it is basically just asking for problems without benefits :)

Reply:
Ok, done it !

Reply:
tx